app/api/views/views.py

```diff
@@ -0,0 +1,135 @@
+        check_name = Validator.validate_login(firstname, lastname, password)
+        if check_name:
+            return check_name
+        users.append(user_data)
+        return jsonify({"message": "Account was created successfully!"}), 201
+
+    @app.route('/api/v1/auth/login', methods=['POST'])
+    def login():
+        info = request.get_json()
+        email = info.get('email', None)
+        password = info.get('password', None)
+        is_valid = validate_email(email)
+        if not is_valid:
+            return jsonify({"error": "invalid email!!"}), 400
+        if not users:
+            return jsonify({'message': 'No users found'})
+        if not email and not password:
+            return jsonify({"message": "enter your credentials"}), 401
+        if len(password) < 8:
+            return jsonify({"error": "invalid password length!!"}), 400
+        if not password:
+            return jsonify({"error": "password field empty!!"}), 400
+        else:
+            return jsonify({'message': "logged in"}), 200
+
+
+class ViewMessage(MethodView):
+
+    @app.route('/api/v1/messages', methods=['POST', 'GET'])
+    def compose_message():
+        if request.method == "POST":
+            info = request.get_json()
+            parentMessageId = info.get("parentMessageId")
+            receiverId = info.get("receiverId")
+            senderId = userid
+            subject = info.get("subject")
+            message = info.get("message")
+            status = "sent"
+            response = Validator.check_value(receiverId, subject, message)
+            if response:
+                return response
+            message_data = {
+                "status": 201,
+                "data": {
+                    "id": len(messages)+1,
+                    "createdOn": str(datetime.now()),
+                    "message": message,
+                    "status": status,
+                    "subject": subject,
+                    "senderId": senderId,
+                    "receiverId": receiverId
+                }
+            }
+            if not senderId:
+                return jsonify(
+                    {"error": "senderID can't be an empty field!!"}), 404
+            else:
+                messages.append(message_data)
+            return jsonify(message_data), 200
+        if request.method == "GET":
+            if not messages:
+                return jsonify({"message": "Inbox is empty!!"})
+            else:
+                return json.dumps(messages)
+
+    @app.route('/api/v1/messages/unread', methods=['GET'])
+    def unread_message():
+        if not messages:
+            return jsonify({"error": "no new mail."}), 404
+        if ['status'] != 'unread':
+            return jsonify({"error": "no mail by  user found!"}), 404
+        return jsonify(messages)
+
+    @app.route('/api/v1/messages/<int:id>', methods=['GET', 'DELETE'])
+    def get_single_message_delete(id):
+            if not messages:
```

Comment:
Lines 116 to 127 are not indented right. Please work on that.

Reply:
okay

Reply:
@mendozabree I have pushed the changes. kindly please find time to review